src/devdox_ai_git/schema/repo.py

```
@@ -0,0 +1,237 @@
+    id: str = Field(..., description="Repository ID from provider")
+    repo_name: str = Field(..., description="Repository name")
+    description: str | None = Field(None, description="Repository description")
+    html_url: str = Field(..., description="Repository URL")
+    relative_path: str = Field(
+        ..., description="The part of the repository's url excluding the domain"
+    )
+
+    default_branch: str = Field(..., description="Default branch name")
+    forks_count: int = Field(..., description="Number of forks")
+    stargazers_count: int = Field(..., description="Number of stars")
+    size: int | None = Field(None, description="Repository size in KB")  # noqa: UP045
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `UP045`)

Remove unused `noqa` directive

(RUF100)